src/lib/autocontainers.ts

```diff
@@ -71,18 +71,22 @@ export class AutoContain implements IAutoContain {
     autoContain = async (
         details: IDetails,
     ): Promise<browser.webRequest.BlockingResponse> => {
+        // Lets not break everyone's user experience ok?
+        let enabled = Config.get("auconenable")
```

Comment:
Please just have `config.get("wherever the autocontains are stored") !== []` or the sane equivalent